src/main/java/net/minestom/server/permission/Permission.java

```diff
@@ -59,6 +59,20 @@ public NBTCompound getNBTData() {
         return data;
     }
 
+    /**
+     * Checks by permission whether the command may be executed or not.
+     *
+     * @param sender
+     * @param commandString
+     * @return whether the sender has the authorization to execute the command.
+     */
+    public boolean matchCondition(@NotNull CommandSender sender, @Nullable String commandString) {
+        if (!(sender.hasPermission(getPermissionName()))) {
+            return false;
+        }
+        return true;
+    }
```

Comment:
You can just `return sender.hasPermission(getPermissionName());` instead of this unnecessary code

Reply:
Also is this method really necessary?

Reply:
ohh i just deleted the noPermissions Message ;) so this code is real stupid xD

> You can just have /permissions <check|add|remove> instead of three different commands

I have to check how Sub Commands works.